packages/cli/src/databases/entities/Project.ts

```diff
@@ -8,8 +8,8 @@ export type ProjectType = 'personal' | 'team' | 'public';
 
 @Entity()
 export class Project extends WithTimestampsAndStringId {
-	@Column({ length: 255 })
-	name: string;
+	@Column({ length: 255, nullable: true })
+	name?: string;
```

Comment:
is there any value in having a default value for the names of personal projects instead of a `null` 🤔 ?

Reply:
Only personal projects should have a null value, and we'll be joining in the user's email on the project list API. This is to avoid having to sync the user's name or email to the project whenever they change it.